freqtrade/plugins/pairlist/ShuffleFilter.py

```diff
@@ -18,8 +19,17 @@ def __init__(self, exchange, pairlistmanager,
                  pairlist_pos: int) -> None:
         super().__init__(exchange, pairlistmanager, config, pairlistconfig, pairlist_pos)
 
-        self._seed = pairlistconfig.get('seed')
-        self._random = random.Random(self._seed)
+        # Apply seed in backtesting mode to get comparable results,
+        # but not in live modes to get a non-repeating order of pairs during live modes.
+        if config.get('runmode') in (RunMode.LIVE, RunMode.DRY_RUN):
+            self._seed = None
+            logger.info("Live mode detected, not applying seed.")
+        else:
+            self._seed = pairlistconfig.get('seed')
+            logger.info(f"Backtesting mode detected, applying seed value: {self._seed}")
+
+        # deprecated since 3.9
+        #self._random = random.Random(self._seed)
```

Comment:
I'm not sure if this is really deprecated this way.

> Deprecated since version 3.9: In the future, the seed must be one of the following types: NoneType, int, float, str, bytes, or bytearray.
[source](https://docs.python.org/3/library/random.html#alternative-generator)

The way i understand this is that it's deprecated to pass in a value that doesn't correspond to one of the given types.

The same message is also on `random.seed()` which we're using now ([here](https://docs.python.org/3/library/random.html#random.seed))

Reply:
You are right, seems like i stopped reading at 3.9.